Corona-Warn-App/src/main/res/layout/fragment_statistics_explanation.xml

```diff
@@ -93,7 +93,10 @@
         <ScrollView
             android:layout_width="0dp"
             android:layout_height="0dp"
-            app:layout_constraintBottom_toBottomOf="@id/guideline_bottom"
+            android:paddingBottom="@dimen/spacing_small"
```

Comment:
Same as above. But there seems to be already more space.

Reply:
The answer is here:

https://github.com/corona-warn-app/cwa-app-android/blob/e1ec946e09600826d16ea389aa8dd96f7c959129/Corona-Warn-App/src/main/res/layout/fragment_statistics_explanation.xml#L431

That's an odd margin… we should remove that before increasing the scroll view's padding.